### PR TITLE
contrib/poweralertd: new package (0.2.0)

### PR DESCRIPTION
**Pull request by handlerug**

### CODE REVIEW

File: contrib/poweralertd/template.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+maintainer = "Umar Getagazov <umar@handlerug.me>"
+license = "GPL-3.0-only"
+url = "https://sr.ht/~kennylevinsen/poweralertd"
+source = f"https://git.sr.ht/~kennylevinsen/poweralertd/archive/{pkgver}.tar.gz"
+sha256 = "f70076dad452c592e2fcdeba4fd533c11394c254f42c21522aa7b56f92a0bd69"
+hardening = ["vis", "cfi"]
+
+
+def post_install(self):
+    self.install_license("LICENSE")
```
Comment (nekopsykose) [inline contrib/poweralertd/template.py:22]: don't have to install gpl

Comment (nekopsykose) [reply contrib/poweralertd/template.py:22]: based on if licence needs distribution, yes for mit

**Comment by q66** [comment]
this one lgtm; we could use basu here for the dbus access, but for now it's ok

**Comment by q66** [comment]
we should have a dinit user service for this but that will need to wait until user services have a graphical.target sentinel (which is coming Real Soon now)

**Pull request merged**